pytorch_lightning/callbacks/stochastic_weight_avg.py

```diff
@@ -222,7 +222,8 @@ def on_train_epoch_start(self, trainer: 'pl.Trainer', pl_module: 'pl.LightningMo
             trainer.num_training_batches += 1
             trainer.fit_loop._skip_backward = True
             self._accumulate_grad_batches = trainer.accumulate_grad_batches
-            trainer.accumulate_grad_batches = len(trainer.train_dataloader)
+
```

Comment:
@justusschock big nit: did you add this newline on purpose? This was a logical block explained by the comment just above